From 83cbb38e1702381acceec472a0733cc3f22cd5ba Mon Sep 17 00:00:00 2001
From: Joel Capitao <jcapitao@redhat.com>
Date: Fri, 11 Mar 2022 17:52:15 +0100
Subject: [PATCH] Remove failing test

---
 tests/middleware_exceptions/tests.py | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/tests/middleware_exceptions/tests.py b/tests/middleware_exceptions/tests.py
index 2a389ce125..fcc60be7fd 100644
--- a/tests/middleware_exceptions/tests.py
+++ b/tests/middleware_exceptions/tests.py
@@ -271,20 +271,6 @@ class MiddlewareSyncAsyncTests(SimpleTestCase):
             'Payment Required: /middleware_exceptions/view/',
         )
 
-    @override_settings(
-        DEBUG=False,
-        MIDDLEWARE=[
-            'middleware_exceptions.middleware.AsyncNoTemplateResponseMiddleware',
-        ],
-    )
-    def test_async_process_template_response_returns_none_with_sync_client(self):
-        msg = (
-            "AsyncNoTemplateResponseMiddleware.process_template_response "
-            "didn't return an HttpResponse object."
-        )
-        with self.assertRaisesMessage(ValueError, msg):
-            self.client.get('/middleware_exceptions/template_response/')
-
     @override_settings(MIDDLEWARE=[
         'middleware_exceptions.middleware.SyncAndAsyncMiddleware',
     ])
-- 
2.35.1

